source/patterns/@aws-solutions-constructs/resources/lib/shared/index.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+import * as lambda from 'aws-cdk-lib/aws-lambda';
```

Comment:
index doesn't seem like the correct name for this file - index.ts usually the top of the hierarchy in some way. This just looks like a utils file?

Reply:
We can switch it to `utils.ts`, made the update in 545e00531f576736467aa50a8bd2654d0dadc80d